src/interface/src/styleguide/filter-dropdown/filter-dropdown.component.html

```diff
@@ -0,0 +1,78 @@
+<button
+  [matMenuTriggerFor]="multiSelectMenu"
+  aria-label="more scenario Options"
+  class="dropdown-button"
+  [disabled]="disabled"
+  (click)="handleFilterClick()"
+  [ngClass]="{ 'active-filter': hasSelections(), disabled: disabled }">
+  <mat-icon *ngIf="leadingIcon"></mat-icon>
+  <div
+    class="dropdown-label"
+    [ngClass]="{ 'active-filter': hasSelections(), disabled: disabled }">
+    {{ selectionText() }}
+  </div>
+  <div *ngIf="showCount()" class="more-items-box">
+    <div class="more-items-icon">&plus;</div>
+    <div class="selected-count-chip">
+      {{ selectedItems.length }}
+    </div>
+  </div>
+  <mat-icon *ngIf="!hasSelections()">keyboard_arrow_down</mat-icon>
+  <mat-icon
+    *ngIf="hasSelections()"
+    class="close-icon"
+    (click)="clearSelections($event)"
+    >close</mat-icon
+  >
+</button>
+
+<mat-menu #multiSelectMenu="matMenu">
+  <div
+    class="search-section"
+    *ngIf="hasSearch"
+    (click)="$event.stopPropagation()">
+    <!-- placeholder for search component -->
+    <sg-input-field leadingIcon="search">
+      <input
+        sgInput
+        type="text"
+        placeholder="Search for {{ menuLabel }}"
+        [(ngModel)]="searchTerm"
+        (ngModelChange)="filterSearch()" />
+    </sg-input-field>
+  </div>
+  <button
+    *ngFor="let item of displayedItems"
+    mat-menu-item
+    role="menuitemcheckbox"
+    aria-checked="isInSelection(item)"
+    (click)="toggleSelection($event, item)">
+    <div class="menu-item-content">
+      <div>{{ item }}</div>
+      <mat-checkbox [checked]="isInSelection(item)"></mat-checkbox>
+    </div>
+  </button>
+
+  <div class="bottom-section">
+    <div class="bottom-content">
+      <div class="clear-all-text" (click)="clearSelections($event)">
+        clear all
+      </div>
+      <div class="button-box">
+        <button
+          mat-stroked-button
```

Comment:
this should probably be our sg-buttons, no? I don't see them matching our styles